.github/workflows/infrastructure.yml

```
@@ -82,61 +115,143 @@ jobs:
-            echo "::notice::TeamCity build triggered successfully (Build ID: $BUILD_ID)"
-          else
-            echo "❌ ERROR: Failed to trigger build"
-            echo "Response: $RESPONSE"
-            exit 1
+      - name: Monitor Build Status (Optional)
+        if: success()
+        continue-on-error: true
+        env:
+          TEAMCITY_URL: "https://grayghost-toolboxai.teamcity.com"
+          TEAMCITY_TOKEN: ${{ secrets.TEAMCITY_PIPELINE_ACCESS_TOKEN }}
+        run: |
```

actionlint (1.7.8)

property "trigger-build" is not defined in object type {}

(expression)